SPECS-EXTENDED/osinfo-db-tools/osinfo-db-tools.spec

```
@@ -0,0 +1,132 @@
+Summary:        Tools for managing the osinfo database
+Name:           osinfo-db-tools
+Version:        1.10.0
+Release:        2%{?dist}
+License:        GPLv2+
```

Same comment as `libosinfo`.

```suggestion
License:        GPL-2.0-or-later
```